src/dashboards/Compact/index.tsx

```diff
@@ -105,7 +105,7 @@ const EnturDashboard = ({ history }: Props): JSX.Element | null => {
         boardId ? getFromLocalStorage(boardId + '-tile-order') : undefined,
     )
 
-    const bikeRentalStations = useBikeRentalStations()
+    let bikeRentalStations = useBikeRentalStations()
```

Comment:
Bruk helst `const` i staden for `let` dersom det ikkje er nødvendig å endre på variabelen.